persistence/persistence-api/src/main/java/org/cbioportal/persistence/util/CustomRedisCachingProvider.java

```diff
@@ -59,17 +59,17 @@
     @Value("${app.name:cbioportal}")
     private String appName;
 
-    @Value("${redis.address}")
-    private String address;
+    //@Value("${redis.address}")
```

Comment:
We need to decided if we are going to run in single server mode, leader follower mode, or allow both options.